index.js

```diff
@@ -148,15 +149,24 @@ exports.disableZoom = (win = activeWindow()) => {
 exports.appLaunchTimestamp = Date.now();
 
 if (is.main) {
-	exports.isFirstAppLaunch = () => {
+	const isFirstAppLaunch = () => {
 		const fs = require('fs');
-		const checkFile = path.join(api.app.getPath('userData'), '.electron-util--has-app-launched');
+		const checkFile = path.join(api.app.getPath('userData'), 'FirstRun', '.electron-util--has-app-launched');
 
 		if (fs.existsSync(checkFile)) {
 			return false;
 		}
 
-		fs.writeFileSync(checkFile, '');
+		try {
+			fs.writeFileSync(checkFile, '');
+		} catch (error) {
+			if (error.code === 'ENOENT') {
+				makeDir.sync(path.join(api.app.getPath('userData'), 'FirstRun'));
```

Comment:
Since we only need to create a directory for `api.app.getPath('userData')`, which is just one directory, you can just use `fs.mkdir`.

Reply:
Thanks for the comments, I've made the updates :)